api/src/main/java/com/exelaration/abstractmemery/services/implementations/FileStorageServiceImpl.java

```diff
@@ -33,4 +33,23 @@ public void save(String fileName, String urlData, String storageLocation) {
   private void createDirectory(String uploadDirectory) {
     new File(uploadDirectory).mkdirs();
   }
+
+  public String getFileURL(String fileName) {
+    String fileURL = "";
+    String storageLocation = "/app/src/main/resources/memes/";
```

Comment:
Isn't this value used in another function in this class?  It should be extracted out to a class variable or constant.